assets/src/components/ResourceChartNarrative.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+function ResourceChartNarrative ({ data, weekRange, gradeSelection, resourceType }) {
+  const resourceData = data.sort((a, b) => b.total_percent - a.total_percent)
+  const unAccessedResources = resourceData.filter(resource => resource.self_access_count === 0)
+  const accessedResources = resourceData.filter(resource => resource.self_access_count >= 1)
+  const narrativeTextResources = {}
+  narrativeTextResources.weekSelection = `Resources accessed from week ${weekRange[0]} to ${weekRange[1]}. `
+  narrativeTextResources.typeOfResource = `Selected resource type(s) ${resourceType.length === 1 ? 'is' : 'are'} ${resourceType}. `
+  narrativeTextResources.gradeFilter = gradeSelection.toUpperCase() !== 'ALL' ? `Filtering on grades ${gradeSelection}.` : 'Getting resources across all grades.'
+  narrativeTextResources.resourcesUnaccessCount = `You have not yet accessed ${unAccessedResources.length} of ${resourceData.length} resources.  `
+  narrativeTextResources.resourcesAccessCount = accessedResources.length !== 0 ? `. You have accessed ${accessedResources.length} of ${resourceData.length} resources.  ` : ''
+  narrativeTextResources.resourcesUnAccessList = unAccessedResources.map(x =>
+    resourceType.length === 1
+      ? `${x.resource_name.split('|')[1]} has been accessed by ${x.total_percent}% of students.`
+      : `${x.resource_name.split('|')[1]} of type ${x.resource_type} has been accessed by ${x.total_percent}% of students.`
+  )
+  narrativeTextResources.resourceAccessList = accessedResources.map(x =>
+    resourceType.length === 1
+      ? `${x.resource_name.split('|')[1]} has been accessed by ${x.total_percent}% of students, and you accessed it ${x.self_access_count} times. The last time you accessed this resource was on ${new Date(x.self_access_last_time).toDateString()}`
+      : `${x.resource_name.split('|')[1]} of type ${x.resource_type} has been accessed by ${x.total_percent}% of students, and you accessed it ${x.self_access_count} times. The last time you accessed this resource was on ${new Date(x.self_access_last_time).toDateString()}`)
+  return (
+    <div id='resource-view-narrative' className='fa-sr-only' aria-live='polite'>
```

Comment:
We may want to set `aria-atomic=true` here since VO seems to only be announcing the changes, not the full text. I think it may be confusing if it's not announced fully, but let me know what you think.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-atomic

Reply:
I do difference when adding this label. Done

Reply:
Okay, cool, we can check this with the accessibility team as well.